indico/modules/events/contributions/controllers/management.py

```diff
@@ -182,7 +183,10 @@ def _process(self):
         contrib_form_class = make_contribution_form(self.event_new)
         form = contrib_form_class(obj=FormDefaults(location_data=inherited_location), event=self.event_new)
         if form.validate_on_submit():
-            contrib = create_contribution(self.event_new, *_get_field_values(form.data))
+            # Creating empty contribution so it can be compared to the created one in flash_if_unregistered
```

Comment:
"the created one" sounds weird. I'd rather go for "the new one" and "Create" instead of "Creating"